### PR TITLE
Scripts and anchors are now on S3

### PR DESCRIPTION
**Pull request by thegoldenmule**

### CODE REVIEW

File: Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        private static readonly string[] _Platforms =
+        {
+            "WebGLPlayer",
+            "WSAPlayerX86"
+        };
+
+        /// <summary>
+        /// Opens the window.
+        /// </summary>
+        [MenuItem("Tools/Application Config Editor %a")]
```
Comment (jsundra) [inline Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs]: I know this keeps with existing naming/organization - but as we add more enklu specific editor tooling and eventually want to act more as a module, we might want to start organizing things under `Enklu/...`. Just a thought!

File: Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+        /// <summary>
+        /// Called when the window is deactivated.
+        /// </summary>
+        private void OnDisable()
+        {
+            if (null != _watcher)
+            {
+                _watcher.Dispose();
+                _watcher = null;
+            }
```
Comment (jsundra) [inline Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs]: Should `UnityEditor.EditorApplication.update -= Update` be here, or is that handled internally with some Unity magic?

File: Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+        /// <summary>
+        /// Finds the index into the platforms array.
+        /// </summary>
+        /// <returns></returns>
+        private int PlatformIndex()
+        {
+            var platform = _config.ParsedPlatform;
+            for (var i = 0; i < _Platforms.Length; i++)
+            {
+                if (_Platforms[i] == platform.ToString())
```
Comment (jsundra) [inline Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs]: Super minor, `platform.ToString()` can be cached outside the loop

File: Assets/Source/Application/Services/EnvironmentUpdateService.cs [hunk 4]
```diff
@@ -2,6 +2,7 @@
 using CreateAR.Commons.Unity.Http;
 using CreateAR.Commons.Unity.Logging;
 using CreateAR.Commons.Unity.Messaging;
+using UnityEngine.Networking.PlayerConnection;
```
Comment (jsundra) [inline Assets/Source/Application/Services/EnvironmentUpdateService.cs]: Unused import?

File: Assets/Source/Player/App/StandardScriptLoader.cs [hunk 5]
```diff
@@ -113,45 +121,72 @@ private IAsyncToken<string> LoadScriptFromNetwork(ScriptData script)
...
             var id = _metrics.Timer(MetricsKeys.SCRIPT_DOWNLOADTIME).Start();
 
-            _http
-                .Download(url)
-                .OnSuccess(response =>
-                {
-                    if (response.NetworkSuccess)
-                    {
-                        _metrics.Timer(MetricsKeys.SCRIPT_DOWNLOADTIME).Stop(id);
+            var req = UnityWebRequest.Get(url);
```
Comment (jsundra) [inline Assets/Source/Player/App/StandardScriptLoader.cs]: Code looks good, but curious why the switch from `IHttpService` to coroutines + `UnityWebRequest`?

File: Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs (hunk 1 above)
Comment (thegoldenmule) [reply Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs]: https://github.com/enklu/enkluplayer/issues/238

File: Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs (hunk 2 above)
Comment (thegoldenmule) [reply Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs]: 👍

File: Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs (hunk 3 above)
Comment (thegoldenmule) [reply Assets/Editor/Tools/EditorWindow/ApplicationConfig/ApplicationConfigEditorWindow.cs]: 👍

File: Assets/Source/Application/Services/EnvironmentUpdateService.cs (hunk 4 above)
Comment (thegoldenmule) [reply Assets/Source/Application/Services/EnvironmentUpdateService.cs]: 👍

File: Assets/Source/Player/App/StandardScriptLoader.cs (hunk 5 above)
Comment (thegoldenmule) [reply Assets/Source/Player/App/StandardScriptLoader.cs]: https://github.com/enklu/commons-unity-http/issues/3

**Pull request merged**